### PR TITLE
[DO NOT MERGE] EOS-22883: [F-8E] S3 HA: fixes and improvements for handling requests during shutdown

### PR DESCRIPTION
**Pull request by SaumitraK531**

### CODE REVIEW

File: s3config-test.yaml [hunk 1]
```diff
@@ -52,6 +52,7 @@ S3_SERVER_CONFIG:                                     # Section for S3 Server
    S3_SERVER_REGION_ENDPOINTS: [s3-us-west-2.seagate.com, s3-us.seagate-test.com, s3-europe.seagate-test.com, s3-asia.seagate-test.com]
    S3_SERVER_DEFAULT_ENDPOINT: s3.seagate-test.com
    S3_SERVER_SHUTDOWN_GRACE_PERIOD: 4                   # S3 server shutdown grace period in seconds, default is 10 seconds.
+   S3_RETRY_AFTER_SEC: 30                               # Retry After in seconds
```
Comment (bkirunge7) [inline s3config-test.yaml:55]: rename to S3_SHUTDOWN_AFTER_SEC

Comment (knrajnambiar76) [reply s3config-test.yaml:55]: 30 second seems high
If failover takes that much time then its ok
Else it can be reduced

**Review by knrajnambiar76 — APPROVED** [review]

**Review by bkirunge7 — APPROVED** [review]

**Comment by SaumitraK531** [comment]
Pre-Merge: http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/2001/
Status: Passed.

**Comment by SaumitraK531** [comment]
Closing this PR.

**Pull request closed without merging**